packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -375,7 +393,14 @@ function ConnectionFallback(props: FallbackProps): JSX.Element {
             </NetworkSelectionContainer>
           </HeaderContent>
 
-          <ConnectionFallbackContainer>
+          <ConnectionFallbackContainer
+            layout="row"
+            imgProps={{
```

Comment:
nit: Rather than passing all the props here, can we directly pass an `img` component? Or pass it as a children?

Reply:
passing an `img` seems like a good idea